experimental/embeddings/scripts/evaluate_document_classification.py

```diff
@@ -4,38 +4,41 @@
 import typer
 from flair.embeddings import TransformerDocumentEmbeddings
 
-from experimental.datasets.hugging_face_dataset import HuggingFaceClassificationDataset
+from embeddings.data.hugging_face_data_loader import HuggingFaceDataLoader
+from embeddings.data.hugging_face_dataset import HuggingFaceDataset
+from embeddings.embedding.flair_embedding import FlairTransformerDocumentEmbedding
+from embeddings.evaluator.text_classification_evaluator import TextClassificationEvaluator
+from embeddings.model.flair_model import FlairModel
+from embeddings.pipeline.standard_pipeline import StandardPipeline
+from embeddings.task.flair_task import FlairTextClassification
+from embeddings.transformation.flair_transformations import ToFlairCorpusTransformation
 from experimental.defaults import RESULTS_PATH
-from experimental.embeddings.tasks.text_classification import FlairTextClassification
 
 app = typer.Typer()
 
 
 def run(
-    embedding: str = typer.Option(..., help="Hugging Face embedding model name"),
+    embedding_name: str = typer.Option(..., help="Hugging Face embedding model name or path."),
     dataset_name: str = typer.Option(...),
```

Comment:
here as above

Reply:
Should we use our model as default or good-performing model as herbert from allegro?